lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -425,7 +426,8 @@ def set_3d_properties(self, zs, zdir):
 
     def do_3d_projection(self, renderer):
         xs, ys, zs = self._offsets3d
-        vxs, vys, vzs, vis = proj3d.proj_transform_clip(xs, ys, zs, renderer.M)
+        vxs, vys, vzs, vis = proj3d.proj_transform_clip(xs, ys, zs,
+                                                        self.axes.M)
```

Comment:
If I am reading this stuff correctly, we could possibly get rid of `renderer` from the argument list of many of these methods (obviously through some sort of deprecation period).

Reply:
Hmm, that is probably true, and shouldn't be too difficult.

Reply:
The last commit does this.